worker/crates/action-sink/src/file/writer.rs

```diff
@@ -152,7 +152,18 @@ fn write_csv(
         .quote_style(csv::QuoteStyle::NonNumeric)
         .from_writer(vec![]);
     let rows: Vec<AttributeValue> = features.iter().map(|f| f.clone().into()).collect();
-    let fields = get_fields(rows.first().unwrap());
+    let mut fields = get_fields(rows.first().unwrap());
+
+    if let Some(ref mut fields) = fields {
+        // Remove _id field
+        fields.retain(|field| field != "_id");
+        // Write header
+        if !fields.is_empty() {
+            wtr.write_record(fields.clone())
+                .map_err(|e| crate::errors::SinkError::FileWriter(format!("{:?}", e)))?;
+        }
```

Comment:
@dongri 
It's not a good suggestion, but in short, it's a suggestion that you can get it even at the beginning in a loop.
Personally, I think it is better to have a high readability than that, because the performance does not change that much.